client/dashboard/app/secondary-menu/style.scss

```diff
@@ -1,3 +1,7 @@
+.dashboard-secondary-menu > * {
+	flex-shrink: 0;
```

Comment:
I'm not sure but below seems to happen if we have more primary menu

<img width="143" height="49" alt="image" src="https://github.com/user-attachments/assets/c23e811e-4b66-4371-b78d-b61cd873e6cd" />

Reply:
The only thing I can think of is to restrict these styles to mobile.

Reply:
Actually the mistake here was that the parent was still shrinking. Forcing it to not shrinks solves a few more problems like needing the logo paddingInlineEnd spacing.